pkg/plugins/builtin/ssh/README.md

```diff
@@ -42,15 +42,52 @@ action:
     ssh_key: '{{.config.mySSHKey}}'
 ```
 
-The output resulting from this configuration will be:
+## Requirements
+
+None by default. Ssh credentials should be retrieved from `{{.config.[sshkey]}}` rather than hardcoded on the template.
```

Comment:
I think [sshkey] is unintuitive here: it looks as if it is a specific syntax while it should be clear it's a placeholder name for a config item.